### DISCUSSION TITLE
Best method to remove dust from image

### DISCUSSION THREAD

**Discussion by cejones1** [ImageMagick/ImageMagick#2440] [Help]
I have some dust on a microscope lens that I cannot remove, but I can take a background image of it as well as an image of my specimen. The dust shows up as a few darker spots in the specimen image as well as the blank background image I take.

I tried to simply minus the blank image from the specimen image, but it seems to inverse the image.

i have tried a number of the mathematically methods to remove the dust but I haven't been able to figure out one that works best. Has anyone else done something similar or have some advice for me?

Thank you for any help,
![background](https://user-images.githubusercontent.com/49353144/90653249-91acf800-e204-11ea-9be9-5fe117640db3.png)
Background image showing 5 smudges caused by particles on my lens.

![specimen](https://user-images.githubusercontent.com/49353144/90653346-abe6d600-e204-11ea-9458-0017d1ca3a94.png)
Specimen image that clearly shows the smudges in the image.  I would like to subtract out the smudges.  Using the surrounding pixels to fill in would work too, but I though I could simply reduce the brightness in the smudge areas to match the surrounding pixels.

**Comment by dlemstra** [comment]
Please don't create duplicate posts I have locked the one in the other repository (https://github.com/ImageMagick/ImageMagick6/discussions/94)

**Reply by cejones1** [reply]
Sorry...  Will do.

**Comment by snibgo** [comment]
For a first approximation, divide the specimen image by the background image:
```
magick microImg.png microFld.png -compose DivideSrc -composite x.png
```
![x](https://user-images.githubusercontent.com/33812368/90661050-a3ab8c80-e23e-11ea-90b2-589a1ed1e4d4.png)

**Reply by cejones1** [reply]
Wow thanks you.. This is awesome
